package.json

```diff
@@ -64,8 +64,7 @@
   "scripts": {
     "postinstall": "./webpack_if_prod.sh",
     "lint": "node ./node_modules/eslint/bin/eslint.js ./static/js",
-    "test": "node ./node_modules/mocha/bin/_mocha --compilers js:babel-register --require static/js/global_init.js 'static/**/*/*_test.js'",
-    "singleTest": "node ./node_modules/mocha/bin/_mocha --compilers js:babel-register --require static/js/global_init.js $1",
+    "test": "./js_test.sh",
```

Comment:
In the background context above I mentioned that `$1` doesn't work in `package.json`, it doesn't replace with anything. npm just appends all arguments to the end of the line